src/Type/Php/MicrotimeFunctionReturnTypeExtension.php

```diff
@@ -26,8 +28,10 @@ public function isFunctionSupported(FunctionReflection $functionReflection): boo
 
 	public function getTypeFromFunctionCall(FunctionReflection $functionReflection, FuncCall $functionCall, Scope $scope): Type
 	{
+		$stringType = new IntersectionType([new StringType(), new AccessoryNonFalsyStringType()]);
```

Comment:
not sure it will be helpful, but I think we could in addition make it a `AccessoryLowercaseStringType`

Reply:
Why? It is neither LC or UC, thus it should be accepted also by `uppercase-string` type.